src/schematools/importer/base.py

```diff
@@ -95,6 +95,25 @@ def resolve(self, source: dict) -> Any:
             return source[self.provenance]
 
 
+class Record(dict):
+    """A single generated row from the importer.
+    Each key matches the database field name, value is the data that should be written.
```

Comment:
There are no `key` and `value` fields. I suppose those were merged into `data`?

Reply:
I've reworded it in #361 - it's about each dict key and value.